WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -376,6 +532,12 @@ private extension SupportTableViewController {
         static let wpHelpCenter = NSLocalizedString("WordPress Help Center", comment: "Option in Support view to launch the Help Center.")
         static let contactUs = NSLocalizedString("Contact Support", comment: "Option in Support view to contact the support team.")
         static let wpForums = NSLocalizedString("WordPress Forums", comment: "Option in Support view to view the Forums.")
+        static let prioritySupportSectionHeader = NSLocalizedString("Priority Support", comment: "Section header in Support view for priority support.")
```

Comment:
We should put identifiers to our localized strings, I remember that being pointed out to me by infra people.